elyra/pipeline/registry.py

```diff
@@ -243,39 +280,18 @@ def build_parameter(self, obj, obj_type):
         data_object['format'] = "string"
         custom_control_id = "StringControl"
         if "type" in obj:
-            data_object['format'] = obj['type']
```

Comment:
I suspect this was unintentionally removed, although the TODO in the airflow-based class implies it might not be necessary.

Reply:
Yeah, I think I will go ahead and add the `data_object['format']` back in. Even though I haven't been able to figure out if it is used anywhere on the frontend, I think it's probably good to have for informational purposes and we may be able to use it in the property descriptions as a type hint to the user. (see https://github.com/elyra-ai/elyra/pull/1620#discussion_r635429036 for more)